uint32 start_lane_id
uint32 end_lane_id
---
uint32[] lane_ids
string message
